### DISCUSSION TITLE
How can I add feed via API

### DISCUSSION THREAD

**Discussion by Hourunze1997** [FreshRSS/FreshRSS#7093] [Q&A] [answered]
How can I add feed via API

**Comment by Alkarex** [comment, marked as answer]
For instance using our GReader API:

https://freshrss.github.io/FreshRSS/en/developers/06_GoogleReader_API.html#google-reader-compatible-api

See the `/reader/api/0/subscription/edit` or the `/reader/api/0/subscription/quickadd` endpoints.

The `quickadd` method is likely the easiest. See for instance a compatible documentation:

* https://feedhq.readthedocs.io/en/latest/api/reference.html#subscription-quickadd
* https://www.inoreader.com/developers/add-subscription
* https://github.com/theoldreader/api/blob/master/README.md#adding-subscription

**Reply by Hourunze1997** [reply]
thank you for your answer.
